doc/source/examples/erds_maps.md

```diff
@@ -1,11 +1,59 @@
 # Compute and visualize ERDS maps
 
-Based on [this MNE-Python example](https://mne.tools/dev/auto_examples/time_frequency/time_frequency_erds.html).
+Based on [this MNE-Python example](https://mne.tools/dev/auto_examples/time_frequency/time_frequency_erds.html), which also provides a bit of background information on ERDS.
 
-## Download data sets
+## Download required files
 Download data files for runs 6, 10, and 14 of subject 1 from the [EEG Motor Movement/Imagery Dataset](https://physionet.org/content/eegmmidb/1.0.0/):
 - [S001R06.edf](https://physionet.org/files/eegmmidb/1.0.0/S001/S001R06.edf?download)
 - [S001R10.edf](https://physionet.org/files/eegmmidb/1.0.0/S001/S001R10.edf?download)
 - [S001R14.edf](https://physionet.org/files/eegmmidb/1.0.0/S001/S001R14.edf?download)
 
-## TODO
+## Load data sets
+In MNELAB, go to `File -> Open...`, select the three downloaded files and click `Open`.
+In the sidebar, the three data sets are now visible.
+The data set loaded last (`S001R14`) is highlighted, indicating it is currently "active".
+Therefore, the info panel shows information about that data set.
+
+## Concatenate data
+Now we want to concatenate the three data sets, starting with `S001R06`.
+In the sidebar, select `S001R06`, then go to `Edit -> Append data...`.
+Select `S001R10` and `S001R14` in the `Source` panel - to select multiple items, hold <kbd>Ctrl</kbd> while clicking.
+Drag the selected data sets to the `Destination` panel and confirm with `OK`.
+A new entry appears in the sidebar: `S001R06 (appended)`.
+The entries `Samples` and `Length` in the info panel confirm that it is the result of the concatenation.
+To close the original data sets, select them in the sidebar and go to `File -> Close` (or press <kbd>Ctrl</kbd>+<kbd>F4</kbd>).
```

Comment:
On macOS, the shortcut is <kbd>⌘</kbd>+<kbd>w</kbd>.

Reply:
Oh, then I'd not mention shortcuts at all, people will see them in the menu anyway.